docs/sphinx/source/use_cases/tensorflow-via-onnx.ipynb

```diff
@@ -0,0 +1,1540 @@
+       "1  1  The Manhattan Project and its atomic bomb help...\n",
+       "2  2  Essay on The Manhattan Project - The Manhattan...\n",
+       "3  3  The Manhattan Project was the name for a proje...\n",
+       "4  4  versions of each volume as well as complementa..."
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f4be6af3-8cfe-440f-ac90-51f3b0cce802",
+   "metadata": {},
+   "source": [
+    "Feed the `data` to the application."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "id": "a0521de2",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Successful documents fed: 10/10.\n",
+      "Batch progress: 1/1.\n"
+     ]
+    }
+   ],
+   "source": [
+    "result = app.feed_df(df=data, include_id=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eb7df3f0",
+   "metadata": {},
+   "source": [
+    "## Validate Vespa predictions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca2a90f9-1ea9-4737-93df-bf667c2f5f23",
+   "metadata": {},
+   "source": [
+    "Get query from the small dev set to use to validate Vespa TensorFlow predictions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "id": "ed63f2fc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dev_small = ir_datasets.load(\"msmarco-passage/dev/small\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "id": "e9fd5566",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for query in dev_small.queries_iter()[:1]:\n",
+    "    query_text = query[1].replace(\"'\", \"\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "id": "433696e4",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'what is paula deens brother'"
+      ]
+     },
+     "execution_count": 34,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "query_text"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0ec295b-07cf-4047-92ea-57547c8d4094",
+   "metadata": {},
+   "source": [
+    "The code below shows the YQL expression that will be used to select the documents to be ranked."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 35,
+   "id": "6ee901a2",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "\"select * from sources * where ({'grammar': 'any', 'defaultIndex': 'text'}userInput('what is paula deens brother'));\""
+      ]
+     },
+     "execution_count": 35,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "\"select * from sources * where ({{'grammar': 'any', 'defaultIndex': 'text'}}userInput('{}'));\".format(query_text)"
```

Comment:
The YQL can be simplified to:
```suggestion
    "\"select * from sources * where ({{grammar: 'any', defaultIndex: 'text'}}userInput('{}'))\".format(query_text)"
```
(can simplify the code below with the same as well)